app/src/components/governance/ParticipantList.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+  <div class="participant-container">
+    <h4>{{ title }}</h4>
+    <ul>
+      <li
+        v-for="(participant, index) in participants"
+        :key="index"
+        class="participant"
+      >
+        <div class="first-column">
+          <span class="icon">
+            <img src="https://lunie.fra1.digitaloceanspaces.com/polkadot.png" />
+            <!-- {{ participant.icon || `n/a` }} -->
+          </span>
+          <span class="name">{{ getParticipantName(participant) }}</span>
+        </div>
+        <span class="voter">{{ participant.address | formatAddress }}</span>
+        <div>
+          <span v-if="!isProposalsPage && participant.option" class="option">{{
+            participant.option
+          }}</span>
+        </div>
+        <div>
+          <span
+            v-if="
+              !isProposalsPage &&
+              participant.amount &&
```

Comment:
if this exists, it should have amount and denom right? so you can reduce the 2 statements line 28 and 35